packages/edit-site/src/experiments.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+import { dispatch } from '@wordpress/data';
+import { createRoot, StrictMode } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import './hooks';
+import { store as editSiteStore } from './store';
+
+/**
+ * Internal dependencies
+ */
+import ExperimentsApp from './components/experiments-app';
+
+/**
+ * Initializes the "Posts Dashboard"
+ * @param {string} id       ID of the root element to render the screen in.
+ * @param {Object} settings Editor settings.
+ */
+export function initializeExperiments( id, settings ) {
+	if ( ! globalThis.IS_GUTENBERG_PLUGIN ) {
+		return;
+	}
+	const target = document.getElementById( id );
+	const root = createRoot( target );
+
+	dispatch( editSiteStore ).updateSettings( settings );
```

Comment:
Probably don't need this.
```suggestion
```

Reply:
Actually, I remember why it was needed not 🤦 The Back button link to the main wp dashboard uses the settings store to get the URL of the site it should navigate to. So will need to get that added back